en/components/overlay_styling.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Overlay Styling
+_description: A detailed walkthrough that explains how to properly apply and scope styles to elements that are displayed using the IgniteUI for Angular Overlay Service.
+_keywords: Ignite UI for Angular, Angular Overlay Service, Angular UI controls, Overlay Service, View Encapsulation Example, Sass scoped styles in Angular, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library
+---
+
+## Introduction
+<p class="highlight">
+
+[`IgxOverlayService`](overlay_main.md) dynamically inserts elements in the DOM. In this topic, we'll go over styling those elements (and components that use the overlay).
+</p>
+<div class="divider--half"></div>
+
+## Premise
+Let's review the following example - we want to display two [`igx-drop-down`](drop_down.md) components in one view. We want one of them to be styled with a custom theme and the other to use the default theme.
```

Comment:
I feel like this premise needs to be a more concise as to not distract from the overlay-specific stuff.
A single drop down (or better yet something without content projection like a combo) should be enough to provide a guide for basic styling. The main goal is to explain the effects of the service moving content on the styles.

_Also scoping the modal style should be a more advanced use - there's a big caviat that should be explained in the main topic about using modal with outlet and the issues that can have with viewport-altering CSS properties._

This is one of the few topics that probably deserve a custom diagram explaining what's going where.